### PR TITLE
Adds an examine hint to explorer gas masks

### LINKED ISSUE #69644: Explorer gas mask does not fit in survival box despite spawning in survival box
Reporting client version: 514.1585

## Round ID:
[189664](https://scrubby.melonmesa.com/round/189664)

## Testmerges:
- [Buffs Heretic Curses, Living Heart, Leeching Walk, and technically Entropic Plume by fixing some bugs](https://www.github.com/tgstation/tgstation/pull/69181)
- [Moves "catch this var/flag" code from obj/init and datum/new into the types that use it ](https://www.github.com/tgstation/tgstation/pull/69634)
- [Macro optimizes SSmapping saving 50% ](https://www.github.com/tgstation/tgstation/pull/69632)

## Reproduction:

1. Spawn in as a miner or acquire a mining conscription kit.
2. Remove Explorer gas mask from Mining Survival box.
3. Attempt to place explorer gas mask back in the survival box it was found in.
4. Observe "The explorer gas mask is too big for the survival box!" displayed as the gas mask does not fit in the survival box it spawned in.

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Closes #69644 

- Explorer gas mask in shaft miner kit spawns folded
- Gas mask adjusted w_class follows initial setup (adjusted = small, unadjusted = normal)
- Adds examine tip to explorer gas masks

## Why It's Good For The Game

You always could fit these in the box, they just had to be adjusted a certain way
So, an examine hit and a bit of a consistency pass should help 

## Changelog

:cl: Melbert
qol: Add an examine tip explaining how to fit an explorer gas mask into a box
/:cl:

### CODE REVIEW

**Comment by msgerbs** [comment]
Not sure I understand why explorer gas masks can't fit in a box unless folded even though regular gas masks can fit fine. Seems pointless to me.

**Comment by MrDoomBringer** [comment]
This should be a screentip instead :-)
Something simple like 'Alt-Click to fold up'
Man we should really have better sprites for the folded state

File: code/modules/mining/equipment/explorer_gear.dm [hunk 1]
```diff
@@ -63,8 +63,18 @@
...
-	w_class = mask_adjusted ? WEIGHT_CLASS_NORMAL : WEIGHT_CLASS_SMALL
+	. = ..()
+	// adjusted = out of the way = smaller = can fit in boxes
+	w_class = mask_adjusted ? WEIGHT_CLASS_SMALL : WEIGHT_CLASS_NORMAL
+
+/obj/item/clothing/mask/gas/explorer/examine(mob/user)
+	. = ..()
+	if(mask_adjusted || w_class == WEIGHT_CLASS_SMALL)
+		return
+	. += span_notice("You could fit this into a box if you adjusted it.")
```
Comment (JohnFulpWillard) [inline code/modules/mining/equipment/explorer_gear.dm:74]: Do they have screentips already? If not, would it be good to add?

File: code/modules/mining/equipment/explorer_gear.dm [hunk 2]
```diff
@@ -63,8 +63,18 @@
...
+
+/obj/item/clothing/mask/gas/explorer/folded
```
Comment (JohnFulpWillard) [inline code/modules/mining/equipment/explorer_gear.dm:76]: Forgot to ask, does this spawn with the mask adjusted?

File: code/modules/mining/equipment/explorer_gear.dm (hunk 1 above)
Comment (MrDoomBringer) [reply code/modules/mining/equipment/explorer_gear.dm:74]: Remove the action button and add a screentip, action button spam is not the way imo

**Pull request merged**